dockerfiles/apache.Dockerfile

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020     Red Hat, Inc.
```

Comment:
new file, new copyright date. I know this is boilerplate, but please set a valid start date of 2021.